nbgrader/tests/test_nbextension.py

```diff
@@ -12,28 +15,46 @@
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.common.keys import Keys
 
+from nbgrader.install import install
+
+
 class TestCreateAssignmentNbExtension(object):
 
     @classmethod
     def setup_class(cls):
         cls.tempdir = tempfile.mkdtemp()
         cls.ipythondir = tempfile.mkdtemp()
+        cls.ipythondir = os.path.expanduser('~/eraseme/temp')
```

Comment:
There are now two `ipythondir`s here -- which one is the one you want to keep? Does it work now if you use the tempdir created from `tempfile`?